screen2.0/src/app/applets/gwas/page.tsx

```diff
@@ -497,22 +497,28 @@ export default function GWAS() {
   }, [plotData, study, suggestionsOpen])
 
   return (
-    <main style={{ backgroundColor: background }}>
+    (<main style={{ backgroundColor: background }}>
```

Comment:
there is an extra "(" and ")" here.

Reply:
Hmm weird, the codemod to fix the grid props must have added these parentheses